packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -189,6 +191,9 @@ exports.sourceNodes = async (
     unstable_createNodeManifest,
   } = actions
 
+  // Set request timeout value from the plugin options.
+  GATSBY_SOURCE_DRUPAL_REQUEST_TIMEOUT = requestTimeoutMS
```

Comment:
that's the reason why we should move it to the payload because if you pass the global variable to a function it won't pass the reference but its value leads to weird expectations on refactoring.